ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -62,8 +95,15 @@ public static long ForSum(int[] values)
         /// <returns>The sum of the numbers.</returns>
         public static long ForEachSum(int[] values)
         {
-            // TODO: Fix this method so it passes all associated tests. A foreach loop must be used to calculate the sum.
-            return -1;
+            // Holds the sum total.
+            int total = 0;
+
+            // A foreach loop that calculates the sum.
+            foreach (int numberIndex in values){
```

Comment:
N14: Ensure that you vertically align curly braces rather than placing them at the end of the line. (-1)